x/proof/keeper/msg_server_create_claim.go

```diff
@@ -52,6 +42,20 @@ func (k msgServer) CreateClaim(
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
+	// Defer telemetry calls so that they reference the final values the relevant variables.
+	defer func() {
+		// Only increment these metrics counters if handling a new claim.
+		if !isExistingClaim {
+			serviceId := session.Header.ServiceId
+			applicationAddress := session.Header.ApplicationAddress
+			supplierOperatorAddress := msg.GetSupplierOperatorAddress()
+
+			telemetry.ClaimCounter(types.ClaimProofStage_CLAIMED, 1, serviceId, applicationAddress, supplierOperatorAddress, err)
+			telemetry.ClaimRelaysCounter(types.ClaimProofStage_CLAIMED, numRelays, serviceId, applicationAddress, supplierOperatorAddress, err)
```

Comment:
Is `ClaimProofStage_CLAIMED` the correct label here?

Reply:
LGTM